lib/deck.rb

```diff
@@ -0,0 +1,29 @@
+require 'card'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.